From d6064ccc491bd3635e397349f9880e6ac21fc142 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 17 Sep 2019 15:15:27 +0000
Subject: [PATCH 355/462] MANTIS 0046652 Java-Build: fix xmlbeans Mantis
 0055465: CUSS 1.5.1 Biometrics Interface

create new projects for CUSS XSD files
xmlbeans has mixed together the generated files from the schema files
added CUSS.1.5.BIOMETRICS

git-svn-id: svn://localhost/SelfServiceBuild/trunk@649 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 2 ++
 SelfServiceBuild/Build/java/build.xml         | 6 ++++++
 2 files changed, 8 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 88881af..6eec9ab 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -238,6 +238,8 @@
 <property name="JarCUSSSBD"                   value="${JarDir}/CUSS.SBD.XSD.jar"/> 
 <property name="JarCUSSIllumination"          value="${JarDir}/CUSS.ILLUMINATION.XSD.jar"/>
 <property name="JarCUSSBiometrics"            value="${JarDir}/CUSS.BIOMETRIC.XSD.jar"/> 
+<property name="JarCUSS15Biometrics"          value="${JarDir}/CUSS-1.5.BIOMETRICS.XSD.jar"/> 
+<property name="JarCUSSSynthesis"             value="${JarDir}/Synthesis.XSD.jar"/> 
 <property name="JarMassaiConfig"              value="${JarDir}/massai-config.jar"/>
 <property name="JarMassaiControlCenter"       value="${JarDir}/massai-controlcenter.jar"/>
 <property name="JarMassaiControlCenterGate"   value="${JarDir}/massai-controlcenter-gate.jar"/>
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 65167eb..2ccd19b 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -27,6 +27,12 @@
     <property name="PlatformDirList" value="
       ${PlatformJavaDir}/ExtractBilling,
       ${PlatformJavaDir}/CUSS,
+      ${PlatformJavaDir}/CUSS.1.5.BIOMETRICS,
+      ${PlatformJavaDir}/CUSS.BIOMETRIC,
+      ${PlatformJavaDir}/CUSS.ILLUMINATION,
+      ${PlatformJavaDir}/CUSS.PAYMENT,
+      ${PlatformJavaDir}/CUSS.SBD,
+      ${PlatformJavaDir}/CUSS.SYNTHESIS,
       ${PlatformJavaDir}/Ingenico-Tempus,
       ${PlatformJavaDir}/OPI.1.3,
       ${PlatformJavaDir}/OPI_CCV20120201,
-- 
2.41.0.windows.1

